blog_post/dbt_blog_post.md

```diff
@@ -0,0 +1,163 @@
+dbt-powered data landscape would not make sense, with both tools keeping track of metadata (e.g.
+data policies) and integrating with data platforms (creating tables or views). Therefore, we soon
+concluded to let dbt be responsible for the creation and updating of new relations (PACE's SQL
+views), and reduce the scope of PACE to the generation of platform-specific SQL based on data policy
+yaml specifications. All while minimizing the impact on a dbt user's existing workflow.
+
+To this end, we started by modularizing PACE, extracting the SQL generation into a `core` module,
+and putting everything related to running PACE as a Spring Boot application in a `server` module.
+This opened the way to adding a `dbt` module, which would know how to generate dbt models
+for PACE data policies, packaged and runnable as a simple executable. The dbt module works as
+follows:
+
+1. A user adds PACE field transforms/filters to their existing `schema.yml` files, on model or
+   column level, under dbt's `meta` sections (see the example below).
+2. After a `dbt compile` or `dbt run`, dbt's `target/manifest.json` file is updated as usual,
+   containing the project's complete configuration, including the PACE policy specifications.
+3. The user runs the PACE dbt module (currently a java jar file) from the dbt project's root, which
+   results in the creation of additional model files, each containing the platform-specific SQL that
+   implements the policy.
+4. During the next `dbt run` execution, dbt creates the secure PACE views on the target platform.
+
+With this approach, the data access policies reside as closely as possible to their respective model
+metadata, becoming an integral part of the dbt project and workflow. Since both the policy
+definitions and their implementations (the PACE models) are part of the same (version-controlled)
+code base, the data access management process is transparent and explicit.
+
+## Example
+
+Let's look at a basic example with an imaginary "transactions" model. Here's the (
+minimal) `schema.yml` file we'll enrich with a PACE data policy:
+
+```yaml
+version: 2
+
+models:
+    -   name: dim_transactions
+        columns:
+            -   name: transactionid
+            -   name: userid
+            -   name: email
+            -   name: age
+            -   name: transactionamount
+            -   name: brand
+```
+
+Let's say that we want to protect the data in the following way:
+
+* The `userid` should be nullified for all users, except for administrators.
+* The `email` should be masked to only show the domain, except for administrators.
+* Administrators can see all records, but everyone else can only see records for users with an `age`
+  greater than 18.
+
+Then we can add the field transforms to their respective column metadata, and the filter rule to the
+model's metadata:
+
+```yaml
+version: 2
+
+models:
+    -   name: dim_transactions
```

Comment:
Should I have a look at the indentation? You had some issues with your IDE you said?

Reply:
No, we have to do it in ghost.io anyway, stuff like that we can fix then.